canopy/app/src/App.js

```diff
@@ -14,80 +14,74 @@
  * limitations under the License.
  */
 
-import React, { useRef, useLayoutEffect, useState } from 'react';
+import React, { useRef, useState, useEffect } from 'react';
 import { loadAllSaplings } from './loadSaplings';
+import { useUserState, UserProvider } from './UserStore';
```

Comment:
You can just say ` import { useUserState, UserProvider } from 'UserStore'; ` here because of the absolute import config

Reply:
✅